bb.edn

```diff
@@ -98,4 +101,10 @@
    ;; TODO: Address vars tagged with cleanup-todo. These
    ;; are left mostly because they are not high priority
    ;; or not well understood
-   :metadata-exceptions #{:large-vars/cleanup-todo}}}}
+   :metadata-exceptions #{:large-vars/cleanup-todo}}
+
+  :ns-docstrings
+  {:paths ["src/main"]
```

Comment:
@cnrpman @andelf Would one of you be interested in adding ns docstrings for `src/electron` in a followup PR? If so, add the path here to enable linting for it